--- uuid-2.3.7/lib/uuid.rb~	2013-04-23 23:40:02.000000000 +0300
+++ uuid-2.3.7/lib/uuid.rb	2013-04-23 23:47:49.576134643 +0300
@@ -63,7 +63,7 @@
 
   # Version number.
   module Version
-    version = Gem::Specification.load(File.expand_path("../uuid.gemspec", File.dirname(__FILE__))).version.to_s.split(".").map { |i| i.to_i }
+    version = "@VERSION@".to_s.split(".").map { |i| i.to_i }
     MAJOR = version[0]
     MINOR = version[1]
     PATCH = version[2]
